editor.planx.uk/src/@planx/components/Question/Public/Question.fixture.tsx

```diff
@@ -1,28 +0,0 @@
-import React from "react";
-
-import Question from "./Question";
-
-export default (
```

Comment:
I'm dropping this `fixture` file as I believe it's unused and not a component pattern we follow elsewhere - any objections? Dropping this also allows to eliminate `Public/package.json`